Formula/riff.rb

```diff
@@ -24,11 +24,18 @@ def install
     end
   end
 
+  def caveats
```

Comment:
Please remove these caveats, as these are not Homebrew-specific. See https://github.com/Homebrew/homebrew-core/pull/43053#issuecomment-521178021.

Reply:
Fixed, thanks!